SAP/spartacus
# GH-8615 Fix: Sort unit list in forms

closes #8615 

![Screen Shot 2021-01-04 at 1 21 07 PM](https://user-images.githubusercontent.com/72991814/103566267-b7732a80-4e8f-11eb-9110-5deafb96b13e.png)

QA Steps:

- Go to `My Company`
- Open `Units` and add or edit a field
- Check that the `unit` dropdown is sorted
- Check same for Budgets, Cost Centers, Users, Purchase Limits, and User Groups
